module/x/gravity/keeper/msg_server.go

```diff
@@ -11,31 +11,16 @@ import (
 	"github.com/cosmos/gravity-bridge/module/x/gravity/types"
 )
 
-type msgServer struct {
```

Comment:
Dont we want to keep this to avoid overlap of method names on the keeper itself?

Reply:
there's no overlap :)